vnext/local-cli/config/configUtils.js

```diff
@@ -0,0 +1,144 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ * @format
+ */
+// @ts-check
+
+const fs = require('fs');
+const path = require('path');
+const glob = require('glob');
+
+const xmldoc = require('xmldoc');
+
+// find VS files matching the pattern
+function findFiles(folder, filenamePattern) {
+  const files = glob.sync(path.join('**', filenamePattern), {
+    cwd: folder,
+    ignore: [
+      'node_modules/**',
+      '**/Debug/**',
+      '**/Release/**',
+      '**/Generated Files/**',
+      '**/packages/**',
+    ],
+  });
+
+  return files;
+}
+
+// finds the windows folder if present
+function findWindowsFolder(folder) {
+  const winDir = 'windows';
+  const joinedDir = path.join(folder, winDir);
+  if (fs.existsSync(joinedDir)) {
+    return joinedDir;
+  }
+
+  return null;
+}
+
+// find the visual studio solution file
+function findSolutionFile(winFolder, packageJson = {}) {
+  if (packageJson.name) {
+    const solutionName = packageJson.name;
+
+    // First look for a solution named solutionName.sln
+    const slnFile = findFiles(winFolder, solutionName + '.sln')[0];
```

Comment:
actually we used to do this in the CLI and just choose the first one, and that was causing issues because playground has both a win32 and uwp SLN in the same folder. I updated the code to bail out if there is more than one SLN since it's not predictable what solution you'd end up building. This can be overridden with the `--sln`  command line argument.
Let's consolidate this code and that to use the same code?

Reply:
Fixed.